examples/cms-contentstack/components/cover-image.js

```diff
@@ -0,0 +1,25 @@
+import cn from 'classnames'
+import Link from 'next/link'
+
+export default function CoverImage({ title, url, slug }) {
+  const image = (
+    <img
```

Comment:
Can this use next/image? 🙏

Reply:
hey @leerob, I have tried implementing the Image component but due to the inline styling of the component. It is breaking CSS structure.
![Screenshot 2](https://user-images.githubusercontent.com/41462986/116243359-81322180-a784-11eb-969e-e29d0cc249aa.png)
![Screenshot 1](https://user-images.githubusercontent.com/41462986/116243380-84c5a880-a784-11eb-9acf-5e3d56593ab8.png)
I have tried many things but not able to resolve it can you help me with this?

Reply:
hey @leerob, can you help us with it?